spec/json_validation_spec.rb

```diff
@@ -24,9 +25,10 @@
       end
 
       json_data = JSON.load_file(json_file)
+      schema = JTD::Schema.from_hash(json_schemas.first)
```

Comment:
Call `schema.verify` to ensure the correctness.

Reply:
This makes sense. One more question. Should I format JTD or leave it as is now?

Reply:
It's fine to leave it as is for now. Ideally, we should have the CI enforce some format though.